src/neo-vm/Types/Array.cs

```diff
@@ -105,5 +105,10 @@ public void Reverse()
         {
             _array.Reverse();
         }
+
+        internal override ReadOnlyMemory<byte> ToMemory()
+        {
+            throw new NotSupportedException();
```

Comment:
Implement it as `virtual` with this content by default in `StackItem`?

Reply:
It's `internal`. So it is the same anyway.